v0/util/index.js

```diff
@@ -590,7 +590,7 @@ const getFieldValueFromMessage = (message, sourceKey) => {
 // - - template : need to have a handlebar expression {{value}}
 // - - excludes : fields you want to strip of from the final value (works only for object)
 // - - - - ex: "anonymousId", "userId" from traits
-const handleMetadataForValue = (value, metadata, integrationsObj = null) => {
+const handleMetadataForValue = (value, metadata, destKey, integrationsObj = null) => {
```

Comment:
```suggestion
const handleMetadataForValue = (value, metadata, integrationsObj = null, destKey) => {
```

Reply:
<img width="394" alt="Screenshot 2022-08-29 at 7 09 29 PM" src="https://user-images.githubusercontent.com/60211312/187214559-439bd1eb-4b6f-4341-8830-7b53de66bb81.png">
Showing sonarlint error

Reply:
Okay, let it be then.